apps/web/src/services/asset-inventory/collector/modules-old/EditScheduleModal.vue

```diff
@@ -318,7 +316,7 @@ export default {
         };
         const addSchedule = async () => {
             try {
-                const params: ScheduleAddParameter = {
+                const params: any = {
```

Comment:
Question: Why isn't the type specified here? Can it accept values of multiple types?🧐

Reply:
Oh this file is gonna be deprecated!